Mathlib/LinearAlgebra/TensorProduct/Basic.lean

```diff
@@ -1486,3 +1527,5 @@ theorem rTensor_neg (f : N →ₗ[R] P) : (-f).rTensor M = -f.rTensor M := by
 end LinearMap
 
 end Ring
+
+set_option linter.style.longFile 1700
```

Comment:
What's this?
Should this be here or somewhere in nolint

Reply:
It needs to be added (at the end of the file, e.g. [here](https://github.com/leanprover-community/mathlib4/blob/16c9d8c2fba7254dbeb6e41eeb0554c23e5776ec/Mathlib/Data/List/Basic.lean#L2319)) when #lines exceeds 1500